### DISCUSSION TITLE
2 RGB LED Strips

### DISCUSSION THREAD

**Discussion by clrmsc** [arendst/Tasmota#10724] [Q&A] [answered]
Hi all! Tasmota is great work!!! 
How to make one module control 2 RGB LED strips? 
I control 1 strip for this mqtt command 

            "type": "lightbulb-HSV",
            "name": "Tasmota",
            "topics": {
                "setHSV": "cmnd/Tasmota/HSBColor",
                "getHSV": "stat/Tasmota/HSBColor",
                "getOn": "stat/Tasmota/Power",
                "setOn": "cmnd/Tasmota/Power"
            },
            "accessory": "mqttthing"

**Comment by barbudor** [comment, marked as answer]
Hi, Tasmota implements a single Light module per device. So you can control only 1 strip per device.

**Reply by sfromis** [reply]
While two RGB strips are indeed not supported, using [SetOption37](https://tasmota.github.io/docs/Commands/#setoption37) you can have "independent handling of RGB and white channels", basically one RGB and one CT strip. That CT can technically be any two colors mixed, or a single color.
